api/dp-openapi.yaml

```diff
@@ -1005,6 +1020,13 @@ components:
         enabled:
           type: boolean
 
+    project_stats:
```

Comment:
Should this be `projects_states`? To me, `project`, singular, is about an individual project's stats. (Ditto the descriptions where you're using `project`, singular.

This is just my confusion, as I'm not really 100% clear on when the singular should be used and when the plural.

Reply:
Well, I'm just basing it on the `round_stats` schema name directly below.

And... re singular/plural: would you say 'height stats' or 'heights stats' for (eg) short/average/tall people? 'Income stats' or 'incomes stats'? For me, I'd use singular, but I don't know if there's a strict grammatical rule to follow.

FWIW, the schema name is just used as an internal label within the OpenAPI definition (the `$ref: '#/components/schemas/project_stats'` earlier in the file). It doesn't make any difference to the queries generated by tools like Swagger.

Reply:
Totally fair -- as I said, to a certain extent, it's about my confusion. And the fact that an individual project's stats would look different from stats about a group of projects.